sanic_jwt_extended/jwt_manager.py

```diff
@@ -108,6 +108,9 @@ def _encode_jwt(cls, token_type, payload, expires_delta):
         if isinstance(expires_delta, datetime.timedelta):
             reserved_claims["exp"] = iat + expires_delta
 
+        if "cookies" in cls.config.token_location and cls.config.csrf_protect:
+            reserved_claims["csrf"] = jti
```

Comment:
```suggestion
        if "cookies" in cls.config.token_location and cls.config.csrf_protect:
            reserved_claims["csrf"] = uuid4().hex
```

Reply:
Thank you for great suggestion. I committed again